en/docs/reference/connectors/kafka-connector/kafka-inbound-endpoint-example.md

```diff
@@ -87,13 +83,13 @@ You can download the ZIP file and extract the contents to get the project code.
 
 ## Deployment
 
-1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `Kafka`. Click on `Kafka Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. Copy this .jar file into  <PRODUCT-HOME>/lib folder. 
+1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `Kafka`. Click on `Kafka Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. Copy this .jar file into  `<MI_HOME>/lib` folder. 
 
-2. Copy the exported carbon application to the <PRODUCT-HOME>/repository/deployment/server/carbonapps folder. 
+2. Copy the exported carbon application to the `<MI_HOME>/repository/deployment/server/carbonapps` folder. 
```

Comment:
If the capp is built and exported in the previous step, do we still have to manually copy here.